### DISCUSSION TITLE
How to introduce the Cross Variogram calculation

### DISCUSSION THREAD

**Discussion by Kenichiro-Yoshimi** [GeoStat-Framework/GSTools#278] [Q&A] [answered]
Sorry for the newbie question.

In the [`vario_estimate`](https://geostat-framework.readthedocs.io/projects/gstools/en/stable/generated/gstools.variogram.vario_estimate.html#gstools-variogram-vario-estimate) function, is there any way to calculate **cross variogram**:
$$\gamma_{\ell m}(r_k) = \frac{1}{2 N(r_k)}\sum_{i=1}^{N(r_k)}(z^{(\ell)}(\mathbf{x}_i)-z^{(m)}(\mathbf{x}_i'))^2$$
or

$$\gamma_{\ell m}(r_k) = \frac{1}{2 N(r_k)}\sum_{i=1}^{N(r_k)}(z^{(\ell)}(\mathbf{x}_i)-z^{(\ell)}(\mathbf{x}_i'))(z^{(m)}(\mathbf{x}_i)-z^{(m)}(\mathbf{x}_i'))$$
 instead of semivarogram when two fields $z^{(\ell)}$ and $z^{(m)}$ are given as a list of numpy.ndarray?

**Comment by MuellerSeb** [comment, marked as answer]
Cross-variogram calculation is not supported at the moment. Given multiple fields will not calculate a cross-variogram but the mean variogram across all given fields.
